doc/datasets/index.rst

```diff
@@ -21,46 +21,49 @@ also possible to generate synthetic data.
 General dataset API
 ===================
 
-There are three main kinds of dataset interfaces that can be used to get 
+There are three main kinds of dataset interfaces that can be used to get
```

Comment:
In the future please make sure to avoid unrelated formatting changes like this one. It  makes the diff bigger than needed and adds noise to the review

Reply:
I'm sorry. It happend because of the setting of my editer. I'll disable it.